### PR TITLE
A2-9: "About Us" Page

### PR DESCRIPTION
**Pull request by lrobert142**
Now with more nesting, less silly variable usage and generally more usability!

### CODE REVIEW

File: static/src/assets/scss/components/_staff-card.scss [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 $staff-card-height: 300px;
 
 .image.card.staff {
     height: $staff-card-height;
-}
 
-.image.card.staff a .reveal h1 {
-    top: $staff-image-hover-heading-offset-top;
-}
+    a .reveal {
```
Comment (axrs) [inline static/src/assets/scss/components/_staff-card.scss]: Remove .reveal from classes. It's been destroyed

File: static/src/assets/scss/pages/_index.scss [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
-    margin-bottom: $spacing-huge;
-  }
+    text-align: center;
+
+    h2 {
+        margin-bottom: $spacing-huge;
+    }
+}
+
+.hero {
```
Comment (axrs) [inline static/src/assets/scss/pages/_index.scss]: I've recently moved this out on another branch... But it's back?.?

File: static/src/pages/about.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    </div>
+</section>
+
+<section class="our-goals" data-equalizer data-equalize-on="medium">
+    <div class="row">
+        <h1>Our Vision, Mission &amp; Values</h1>
+
+        <div class="large-6 columns goal" data-equalizer-watch>
+            <h1>Vision</h1>
```
Comment (axrs) [inline static/src/pages/about.html]: Should be `h3` level

File: static/src/pages/about.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="large-4 columns aside">
+            <img src="http://placehold.it/300x300" width="300" height="300" alt="About Us" />
+        </div>
+
+    </div>
+</section>
+
+<section class="our-goals" data-equalizer data-equalize-on="medium">
+    <div class="row">
+        <h1>Our Vision, Mission &amp; Values</h1>
```
Comment (axrs) [inline static/src/pages/about.html]: Should be `h2` level

File: static/src/assets/scss/components/_staff-card.scss (hunk 1 above)
Comment (lrobert142) [inline static/src/assets/scss/components/_staff-card.scss]: Done

**Pull request merged**